LCLSGeneral/LCLSGeneral/POUs/Data/FB_LREALBuffer.TcPOU

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+  <POU Name="FB_LREALBuffer" Id="{bbdb5874-82a3-4838-8bc3-8b225e25cc46}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_LREALBuffer
+(* An example use of FB_DataBuffer for the likely most-common use case. *)
+VAR_INPUT
+	bExecute: BOOL;
+	fInput: LREAL;
+END_VAR
+VAR_OUTPUT
+	arrOutput: ARRAY [1..1000] OF LREAL;
```

Comment:
I'd imagine there's no way we could do this dynamically based on cycle time, right?

Ideally, we'd want just `1 / (cycle time)` entries, to get 1 array update per second. E.g., `1/0.001` -> 1000 elements for 1ms/1KHz

Reply:
Unfortunately we can't dynamically set array sizes. We can make function blocks that handle arrays of variable lengths, but at some point we need to define an array size in a header.

If there is some standard of rate that we want (1 update per second?) we can make a function block that allocates 1000 elements for the cycle-time = 1ms case, and then use subsets of this array for other cycle times. There must be a simple way to query the cycle time, but I don't know what it is.

Reply:
Or we let the user choose the pre-defined FB that matches up with their cycle time setting? In the common varieties of 1ms, 2ms, ... Perhaps not...

Reply:
It's not very hard to write python code to generate a matrix of twincat function blocks of lengths 1000, 500, 250, 200, 100 and of types LREAL, LINT, BOOL. If you take a look at the LREAL/1000 block I wrote it's very jinja-able.